README.md

```diff
@@ -107,61 +99,36 @@ Or you can replace the prompt with anything you'd like.
 
 ## How to use
 
-```sh
-  Run interactive mode
-      t
-        ctrl-s list only tmux sessions
-        ctrl-x list only zoxide results
-        ctrl-f list results from the find command
-
-  Go to session (matches tmux session, zoxide result, or directory)
-      t {name}
-
-  Open popup (while in tmux)
-      <prefix>+T
-        ctrl-s list only tmux sessions
-        ctrl-x list only zoxide results
-        ctrl-f list results from the find command
-
-  Show help
-      t -h
-      t --help
-```
-
 By default, this plugin is bound to `<prefix>+T` which triggers a fzf-tmux popup that display zoxide results. Type the result you want and when you hit enter it will create a tmux session and connect to it or, if the sessions already exists, switch to it.
 
 If you are not in tmux, you can simply run `t` to start the interactive script, or call `t {name}` to jump directly to a session of your choosing.
 
-### Key Bindings
-
-- `ctrl-s` list only tmux sessions
-- `ctrl-x` list only zoxide results
-- `ctrl-f` find by directory
+You can run `t --help` or `t -h` to see the help menu.
 
 ## How to customize
 
-### Use Git Root for session name
-
-You may prefer your session names starting from the root of the git repository. This can help with naming conflicts if you have multiple directories with the same name on your machine and make it clear when you have multiple sessions open in the same git repository.
-
-<details>
-<summary>bash</summary>
-
-Add the following line to `~/.bashrc`
+There are many ways you can customize this script to fit your workflow. I am also open to pull requests if you have ideas for improvements. Customizations are stored as shell variables, so they can be accessed with or without tmux running. Here are the available variables:
 
-```sh
-export T_SESSION_USE_GIT_ROOT="true"
-```
-
-</details>
+- `T_FZF_FIND_BINDING`: overwrites the fzf `ctrl-f` key binding, typically used for finding a directory that's not listed in zoxide.
+- `T_FZF_PROMPT`: overwrites the fzf prompt symbol
+- `T_SESSION_USE_GIT_ROOT`: detects the git root of a chosen directory when generating a tmux sessions name
+- `T_SESSION_NAME_INCLUDE_PARENT`: includes the parent directory in the tmux session name
+- `T_FZF_BORDER_LABEL`: overwrites the fzf popup border label
+- `T_FZF_DEFAULT_RESULTS`: overwrites the default fzf results, can be "sessions" or "zoxide" (defaults to sessions and zoxide)
 
 <details>
-<summary>zsh</summary>
+<summary>zsh or bash</summary>
 
-Add the following line to `~/.zshrc`
+Add the following line to `~/.zshrc` for zsh, and `.bashrc` for bash.
 
 ```sh
-export T_SESSION_USE_GIT_ROOT="true"
+# use fd and search two levels deep from home
```

Comment:
You are adding fd as a dependency without mentioning it in the Prerequisites in the readme